src/Microsoft.Extensions.Localization/Internal/AssemblyResourceStringProvider.cs

```diff
@@ -65,6 +63,7 @@ private IList<string> ThrowOrNull(CultureInfo culture, bool throwOnMissing)
         public IList<string> GetAllResourceStrings(CultureInfo culture, bool throwOnMissing)
         {
             var cacheKey = GetResourceCacheKey(culture);
+
```

Comment:
So is that mean I should revert all the blank lines except after curly braces and before `return`?!! if is that there are many places in this repo where blank line is added after `var` line